.sandcastle/Dockerfile

```
@@ -0,0 +1,40 @@
+# The exact browser binary is installed later by the sandbox hook (after npm install
+# resolves the project's playwright version), but the OS packages must be root-installed.
+RUN npx --yes playwright install-deps chromium
+
+# Rename the base image's "node" user (UID 1000) to "agent".
+# This keeps UID 1000 so that --userns=keep-id (Podman) and
+# --user 1000:1000 (Docker) map to the correct home directory owner.
+RUN usermod -d /home/agent -m -l agent node
+USER agent
+
+# Install Claude Code CLI
+RUN curl -fsSL https://claude.ai/install.sh | bash
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)